### DISCUSSION TITLE
Missing Filter Templates in v4

### DISCUSSION THREAD

**Discussion by svermaak** [microsoft-search/pnp-modern-search#3026] [Ideas]
**Version used**
Modern Search v4 - June 2021

**Describe the bug**
The Filter templates in v4 are missing a whole bunch of options

v3
![image](https://user-images.githubusercontent.com/12259728/124550604-3e6f8280-de74-11eb-9dec-4dde0e702782.png)

v4
![image](https://user-images.githubusercontent.com/12259728/124550781-8c848600-de74-11eb-9b8f-71c3fa9caed8.png)

v3 had option for file type which added icons
![image](https://user-images.githubusercontent.com/12259728/124551004-dff6d400-de74-11eb-8fc6-1e9d3ffdd27f.png)

**To Reproduce**
Create filter type "File Type"

**Expected behavior**
Icons should show

**Screenshots**
See above

**Desktop (please complete the following information):**
Chrome

**Additional context**
None

**Comment by FranckyC** [comment]
Hi @svermaak, these filter types have not been implemented yet so it is not a bug. Also multi/single values are now handled via a dedicated option in the filter configuration. i can't give you any ETA when these templates will be available.

**Comment by svermaak** [comment]
Thank you for your prompt reply, much appreciated Francky

**Comment by Heavm00** [comment]
Hi, is there any update about this issue? It would be great, if the icons for file type would be implemented :-)
Thanks.

**Comment by kasperbolarsen** [comment]
Working on a custom Refiner layout , gotten to this with out any code changes:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/9a07a509-114b-4fa6-addb-fd4c8dce9a5e)


Once I am done I plan to add it to the project

**Comment by MirasMNC** [comment]
Hi, is there any update about this case? I have the same scenario and looking for the solution. It would be the best way. :)

**Reply by kasperbolarsen** [reply]
The file icon Refiner is available at https://github.com/microsoft-search/pnp-modern-search-layouts/tree/main/Refiner/File%20Extensions%20icons

**Reply by MirasMNC** [reply]
@kasperbolarsen thank you for your advice. By any chance, do you know if there is template for multiperson column? I guess it is similar topic here to [this](https://github.com/microsoft-search/pnp-modern-search/discussions/3582). It is quite common use case what we have.

**Reply by kasperbolarsen** [reply]
Not that I know. Multipeople sounds like a endless number of combinations 😊

**Reply by MirasMNC** [reply]
If I understand it right. It is impossible to solve it. I suppose there can be solution if it can be similar case like "Choice" column with more choices and filter should be able to work with it. 😊
